rgbd/src/client.cpp

```diff
@@ -0,0 +1,58 @@
+#include "rgbd/client.h"
+
+#include <opencv2/core/mat.hpp>
+
+#include "rgbd/image.h"
+#include "rgbd/ros/conversions.h"
+
+namespace rgbd {
+
+// ----------------------------------------------------------------------------------------
+
+Client::Client()
+{
+}
+
+// ----------------------------------------------------------------------------------------
+
+Client::~Client()
+{
+}
+
+// ----------------------------------------------------------------------------------------
+
+void Client::intialize(const std::string& server_name, float timeout)
+{
+    if (client_shm_.intialize(server_name, timeout))
+        return;
+
+    client_rgbd_.intialize(server_name);
```

Comment:
I'd prefer to have a consistent API, i.e., that each `Client.initialize` returns a bool that is checked. 

Furthermore, there is no logging here: it might be nice to be able to know which client is used. Personally, I don't see a reason why we shouldn't use ROS logging.

Reply:
Implemented